CMakeLists.txt

```diff
@@ -99,6 +99,18 @@ else()
   set(_FINDPACKAGE_PROTOBUF_CONFIG_DIR "${TRITON_THIRD_PARTY_INSTALL_PREFIX}/protobuf/lib/cmake/protobuf")
 endif()
 
+# CURL is installed to ${TRITON_THIRD_PARTY_INSTALL_PREFIX}/curl/lib64 instead
```

Comment:
Put the code block inside conditional at line 102.

Reply:
If seems like using cmake's find_library and turning off all the default paths may be a better way to do it. https://cmake.org/cmake/help/latest/command/find_library.html

Reply:
Giving find_library a shot. However in this case curl is installed as an external project so maybe find_package would be more accurate for this case?

- [find_library](https://cmake.org/cmake/help/latest/command/find_library.html) to find a library, shared or static
- [find_package](https://cmake.org/cmake/help/latest/command/find_package.html) to find and load settings from an external project.

Reply:
@deadeyegoodwin find_library will not give us the package version. This is set by the find_package call but not by find_library. Are we fine with this change? It would make it harder to catch the version issues like we saw in the issue mentioned in this PR.